README.md

```diff
@@ -87,48 +99,39 @@ integration/continuous development toolchain.
 It's recommended to install and Integrated Developer Environment, or a text editor, to help manage package dependencies, interperters, and overall code execution. There are several options available, some of them are free and some require payment:
 
 * [PyCharm](https://www.jetbrains.com/pycharm/download/) community edition is free, Professional version requires subscription.
+* [Visual Studio Code](https://code.visualstudio.com/Download) free text editor from Microsoft with a wide variety of extensions.
 * [Komodo Edit](https://www.activestate.com/komodo-edit), free, text editor, stripped down version of [Komodo IDE](https://www.activestate.com/products/komodo-ide/features/) (paid version).
 
 ### Setup the Project
-
-
-1. Create a Project Directory:
-    * Create a directory on your filesystem to store the project files. You can create the directory using the IDE toolbar or the command line. Below is an example of using the command line to create your project directory:
-
-    ```
-    $ mkdir python_tests
-    ```
     
-2. Setup `python` interpreter:
+1. Setup `python` interpreter:
     * In your IDE, select an interpreter that references the version of Python installed on your system. Below are links to the relevant documentation for each IDE:
         * [Setup Interpreter in PyCharm](https://www.jetbrains.com/help/pycharm/configuring-python-interpreter.html)
+        * [Setup Interpreter in Visual Studio Code](https://code.visualstudio.com/docs/languages/python)
         * [Setup Interpreter in Komodo IDE](http://docs.komodoide.com/Manual/tutorial/pythontut#python-tutorial-komodo-ide-only_analyzing-the-python-files_setting-up-the-preprocess-py-program_lines-59-to-65-importing-standard-python-modules)
     
     * (Recommended) Use a `virtualenv` to manage dependencies. See the following documentation for more details:
         * [Setup `virtualenv` in PyCharm](https://www.jetbrains.com/help/pycharm/creating-virtual-environment.html)
         * [Setup `virtualenv` in Komodo IDE](http://docs.komodoide.com/Manual/tutorial/pythontut#python-tutorial-komodo-ide-only_analyzing-the-python-files_setting-up-the-preprocess-py-program_lines-59-to-65-importing-standard-python-modules)
         * [Configure `virtualenv` in the command line](https://virtualenv.pypa.io/en/latest/userguide/#usage)
+
```

Comment:
Should we have a requirements.txt file here? That would be conventional in python projects.